src/index.js

```diff
@@ -1,4 +1,72 @@
 import './style.scss';
 
-// You can specify which plugins you need
-// import { Tooltip, Toast, Popover } from 'bootstrap';
\ No newline at end of file
+const leaderBoardApi = document.querySelector('#leaderboard-api');
+
+let gameID = 0;
+async function createGameId() {
```

Comment:
- [ ] As requested by the previous reviewer, kindly refactor your function to use ES6 syntax (arrow function):
### Async arrow functions look like this:
```
const foo = async () => {
  // do something
}
```

Reply:
Well done!